test/rotations.jl

```diff
@@ -62,6 +62,12 @@ include("utils.jl")
         v = log(M, pts[1], pts[2])
         @test norm(M, pts[1], v) ≈ (angles[2] - angles[1]) * sqrt(2)
 
+        # check that exp! does not have a side effect
+        q = copy(M, pts[1])
```

Comment:
We could actually move this test to general manifold tests. It is something `Manopt.jl` assumes so checking it for all manifolds makes sense IMO.

Reply:
`Manopt.jl` would be maybe a little generic, actually _I_ always assumed that that is the case ;)

Reply:
Since you also opened an Issue for the generic case, maybe let‘s solve it in a generic separate PR?

Reply:
OK, sure, I'll try that in a separate PR in the evening today or tomorrow.

Reply:
Hm, we do not have the copy you proposed to use.

Reply:
Weird, I think we should have that `copy`.

Reply:
I agree, we should, since we have `copyto!`, but maybe lets do a new PR (even in ManifoldsBase?) it would just combining existing methods.

Reply:
Sure, it's an issue for `ManifoldsBase.jl`.